apps/storybook-e2e/src/e2e/map/action/click-action/vl-map-click-action.stories-dom.cy.ts

```diff
@@ -0,0 +1,39 @@
+describe('story vl-map-select-actions default', () => {
+    const mapClickActionUrl =
+        'http://localhost:8080/iframe.html?id=map-action-click-action--map-click-action-default&viewMode=story';
+
+    function getMap() {
+        return cy.get('vl-map');
+    }
+
+    function getMapShadowDOM() {
+        return getMap().shadow();
+    }
+
+    it('should render a map', () => {
+        cy.visit(mapClickActionUrl);
+
+        getMapShadowDOM().find('div#map');
+    });
+
+    it('should NOT render with features layers', () => {
+        cy.visit(mapClickActionUrl);
+
+        getMap().find('vl-map-features-layer').should('not.exist');
+    });
+
+    it('should render click-action', () => {
+        cy.visit(mapClickActionUrl);
+
+        getMap().find('vl-map-click-action');
+    });
+
+    it('should NOT render any other map related action(s)-*', () => {
```

Comment:
Deze test vind ik ook wat overbodig. Je kan in principe hier al onze componenten die we hebben aan toevoegen.

Reply:
gedaan - verwijderd